README.adoc

```diff
@@ -159,6 +159,9 @@ The possible options are:
   * `--path`: Download the binary to the specified path on the host.
 +
 The supported options for `service` are: Docker, OpenShift.
++
+Client binaries are downloaded in `data/service-manager/bin/<service>/<cli-version>/` directory under Vagrant home directory by default.
```

Comment:
"Client binaries are per default downloaded to `$VAGRANT_HOME/data/service-manager/bin/<service>/<cli-version>/`. Per default `$VAGRANT_HOME` is `.vagrant.d` in your home directory.